pkg/preflight/run_test.go

```diff
@@ -0,0 +1,219 @@
+//go:build integration
+
+// NOTE: requires a Kubernetes cluster in place currently, hence hidden behind a tag above
+// TODOLATER: get a mocked or ephemeral/Docker based K8s for use? see below some approaches which haven't played out yet
+// Test using: go test --tags=integration
+
+package preflight
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+	/*
+		See TODOLATER below
+
+		"k8s.io/client-go/kubernetes/fake"
+		k8sruntime "k8s.io/apimachinery/pkg/runtime"
+		discoveryfake "k8s.io/client-go/discovery/fake"
+	*/)
+
+func TestRunPreflights(t *testing.T) {
+	t.Parallel()
+
+	// A very simple preflight spec (local file)
+	_, testFilename, _, status := runtime.Caller(0)
```

Comment:
I need to pass in the actual filename, not the content of the file for test purposes here so that function won't cover it

but https://github.com/replicatedhq/troubleshoot/blob/main/internal/testutils/utils.go#L21 does save me some repetition, so I will use that :) thx